### ISSUE TITLE
Bug - 1.19.2 Transparent block faces within the statues

### ISSUE THREAD

**Issue by DracoExE** [TheGrimsey/Statues#2]
Hi there, I just tested the mod in 1.19.2 and I found that the statues makes the block faces they're touching transparent. Thought it was from Iris Shaders, but it happened with shaders off.

Here're some images of the issue
![2022-08-26_15 47 49](https://user-images.githubusercontent.com/44948988/186980853-39c65de1-3280-435d-8660-d27bfffeb973.png)
![2022-08-26_15 48 27](https://user-images.githubusercontent.com/44948988/186980859-88e35eeb-bb73-4b98-b66c-1ec893a70a88.png)

Mod Version: 1.1.1
Fabric Loader: 0.14.9-1.19.2
Fabric API: fabric-api-0.60.0+1.19.2

I'm using other mods, let me know if you want the log with all mods installed (no idea if that coiuld help).

**Comment by TheGrimsey** [comment]
Are you using More Culling?

**Comment by DracoExE** [comment]
> Are you using More Culling?

Yes, I'm using the MoreCulling mod made by FX_PR0CESS (version 0.10.0)

**Comment by TheGrimsey** [comment]
So yeah, this is partially an issue with that. If you disable aggressive blockstate(?) culling it should fix it. Because the statue is a fullblock it assumes it can cull faces.

It's fixable on my end and I will do that soon:tm: as well.
